_sass/minima/_article.scss

```diff
@@ -7,8 +7,11 @@ article.guide {
 
   figure {
     &.-background {
-      padding: 20px;
```

Comment:
Could you please revert the CSS changes? This fix is also needed in #240. I'm addressing it in #261, which we can merge in right away so the fix is available for everyone. Good to keep content and code fixes separate.

Reply:
Reverted

Reply:
Just an FYI shouldn't need to revert anything like this, once @ConorOkus merges #261 into this branch it should be all good. Otherwise, this may create a more recent commit that then undoes the fix 🤕

Reply:
I have no idea how Github will treat this and can't give any recommendations.